packages/web3-providers-ipc/types/index.d.ts

```diff
@@ -20,9 +20,6 @@
  * @date 2018
  */
 
-import * as net from "net";
-import {IpcProviderBase} from 'web3-core-helpers';
-
-export {JsonRpcPayload, JsonRpcResponse} from 'web3-core-helpers';
+import { IpcProviderBase } from 'web3-core-helpers';
```

Comment:
The additional exports can still stay because it isn't any logical change and does improve the TS DX.